anomalib/utils/loggers/tensorboard.py

```diff
@@ -102,4 +102,4 @@ def add_image(self, image: Union[np.ndarray, Figure], name: Optional[str] = None
             image = buffer.reshape(image.canvas.get_width_height()[::-1] + (3,))
             kwargs["dataformats"] = "HWC"
 
-        self.experiment.add_image(img_tensor=image, tag=name, **kwargs)
+        self.experiment.add_image(img_tensor=image, tag=name, dataformats="HWC", **kwargs)
```

Comment:
Why is `dataformats` added? In line 103 we are already setting `dataformats`.

Reply:
Hmm yes I didn't notice that earlier. The weird thing is that it fails due to dimension ordering mismatch when I don't pass this explicitly.

Reply:
Then maybe we can remove line 103